beesdoo_easy_my_coop/models/res_partner.py

```diff
@@ -47,13 +38,8 @@ def _is_worker(self):
             share_type = rec._cooperator_share_type()
             if share_type:
                 rec.is_worker = share_type.allow_working
-                rec.worker_store = share_type.allow_working
             else:
                 rec.is_worker = False
-                rec.worker_store = False
```

Comment:
Shoud I leave `worker_store` for coherency ?

Reply:
nope not needed anymore !